pontoon/base/models.py

```diff
@@ -180,6 +182,30 @@ def user_translated_locales(self):
 
     return [locale.code for locale in locales]
 
+
+@property
+def user_managed_locales(self):
+    locales = get_objects_for_user(
+        self, 'base.can_manage_locale', accept_global_perms=False)
+
+    return [locale.code for locale in locales]
+
+
+@property
+def user_rights(self):
+    if self.is_superuser:
+        return 'Admin'
+    if self.translated_locales or self.managed_locales:
+        output = ''
+        if self.managed_locales:
+            output += 'Manager for ' + ', '.join(self.managed_locales)
+        if self.translated_locales:
+            output += (', ' if self.managed_locales else '') + 'Translator for ' + ', '.join(self.translated_locales)
```

Comment:
We should only say if it's a Manager or Translator, not both (since all Managers are also Translators).